### ISSUE TITLE
Add Eigen and Boost dependencies

### ISSUE THREAD

**Issue by remod** [ethz-asl/ros_best_practices#10]
Adding Eigen and Boost to CMakeLists.txt/package.xml would add help on how to handle these deps correctly.

Do you agree?

**Comment by pfankhauser** [comment]
Yes, I agree, but maybe we should include them but commented out, such that the user can active them if needed.

**Comment by remod** [comment]
See https://github.com/ethz-asl/ros_best_practices/pull/11.
